en/components/general/update-guide.md

```diff
@@ -51,6 +51,44 @@ Unfortunately not all changes can be automatically updated. Changes below are sp
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+## From 16.1.x to 17.0.x
+
```

Comment:
Can you add a paragraph, describing the change from oldSelection/newSelection to oldValue/newValue and also the fact that there are migrations for that?